cfme/fixtures/pytest_selenium.py

```diff
@@ -305,6 +322,14 @@ def wait_for_ajax():
     """
 
     _thread_local.ajax_log_msg = ''
+    orig_events = get_events()
+
+    def _events_not_finished():
+        events = get_events()
+        not_finished = False
+        not_finished |= orig_events["maxTimeoutId"] >= events['minTimeoutId'] > 0
```

Comment:
Sound good but with if please, Id still like to get rid of the chained comparison.